pkg/controllermanager/controller/project/project_control_reconcile.go

```diff
@@ -50,7 +50,7 @@ func (r *projectReconciler) reconcile(ctx context.Context, project *gardencorev1
 
 	// Ensure that we really get the latest version of the project to prevent working with an outdated version that has
 	// an unset .spec.namespace field (which would result in trying to create another namespace again).
-	if err := gardenReader.Get(ctx, kutil.Key(project.Name), project); err != nil {
+	if err := gardenClient.Get(ctx, kutil.Key(project.Name), project); err != nil {
```

Comment:
This get call seems to be superfluous, given that the project is already read from the cache in https://github.com/gardener/gardener/blob/004b5d44976670a9da25017befcda0c205cb31bb/pkg/controllermanager/controller/project/project_control.go#L99-L106